docs/shell-tips.md

```diff
@@ -164,10 +194,134 @@ have the newline character, and thus the `\<newline>`.
 The newline character is invisible (we only see a line break), but it's still
 there!
 
+## Arguments
+
+Most of the commands need additional data to work, like a path or a file. That
+extra information is called an **argument**, and it's specified after the name
+of the command, like this:
+
+```
+$ cd /home/john/notes
+```
+
+Here, the main command is `cd`, and the first (and only) argument is
+`/home/john/notes`.
+
+In each command the arguments are specified in different ways, and have
+different meanings.
+
+Sometimes, a command can accept multiple arguments. In those cases, dashes are
+used to indicate the different arguments. Here's another example:
+
+```
+$ nano -C /home/john/backups --mouse todo.txt
+```
+
+In this case, we're saying: "`nano`, open the file `todo.txt`, enabling mouse
+support, and saving the backup files to `/home/john/backups`". Don't worry, you
+don't have to memorize the meaning of all the arguments for every single
+command. There are [tools](#understanding-commands) that help you with that
+:wink:.
+
+As you can see, some arguments imply that more information has to be specified,
+while others don't:
+
+- `-C` needs you to indicate where the backups should be stored, and thus you
+  have to add an additional argument after it (`/home/john/backups` in the
+  example).
+
+- `--mouse` is just an option you set, `nano` doesn't need anything else to
+  make it work. Thus, there isn't any extra argument for that.
+
+Note that the `todo.txt` is the file we want to open! It has nothing to do with
+the previous argument. This will probably clarify it (taken from `nano`'s
+help):
+
+```
+Usage: nano [OPTIONS] [FILE]...
+```
+
+So, in the options you indicate the arguments, and `FILE` is... well, the file.
+
+Cool, huh?
+
+## Shebangs
+
+You can run some files directly, without specifying a program to interpret
+them.
+
+That's why you may have seen cases when some Python scripts are called with
+`python`:
+
+```
+$ python my_program.py
+```
+
+While other times, `python` isn't used:
+
+```
+$ ./my_program.py
+```
+
+In the latter, it's skipped because `my_program.py` already specifies in it
+which interpreter should be used (in this case, `python`).
+
+This is indicated in the very first line of the script files, and it's called
+a **shebang**. It looks like this, in the case of Python scripts:
```

Comment:
Hmmm, removing `the` here sounds to me like a hypothetical (e.g. "in case of emergency").

Definitely your criteria is much better than mine, but do you mind if we check with a native speaker in Zulip?